resources/js/Components/Form/DateTimePicker.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <ctk-date-time-picker v-model="proxyValue" :minute-interval="minuteInterval" :format="format">
+  <ctk-date-time-picker dusk="date-time-picker" :id="id" v-model="proxyValue" :minute-interval="minuteInterval" :format="format">
```

Comment:
kind of curious why have an ambiguous dusk name here? if there's more than one on the same page then the dusk tag won't work.

Reply:
If was going to be used for the dusk component but we ended up using the id. If someone only has 1 on a page and doesn't want to put an id they can use the dusk tag.